app/src/main/java/ar/com/wolox/android/example/ui/news/NewsAdapter.kt

```diff
@@ -0,0 +1,51 @@
+package ar.com.wolox.android.example.ui.news
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import android.widget.ToggleButton
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.model.New
+
+class NewsAdapter : ListAdapter<New, NewsAdapter.ViewHolder>(NewsDiffUtil()) {
+
+    private lateinit var context: Context
+    private lateinit var view: View
+    var newsList = ArrayList<New>()
+
+    class ViewHolder(view: View) : RecyclerView.ViewHolder(view) {
```

Comment:
First, move this outside adapter and give a more descriptive name. Such as `NewsViewHolder`